Flow.Launcher/SettingWindow.xaml.cs

```diff
@@ -435,6 +435,12 @@ private void OnPluginSettingKeydown(object sender, KeyEventArgs e)
                 pluginFilterTxb.Focus();
         }
 
+        private void LauncherPositionSelectedIndexChanged(object sender, SelectionChangedEventArgs e)
```

Comment:
what's this method for?

Reply:
I don't know either. It seems like a code that VS automatically makes. When I'm making something, I sometimes see this blank code.